MissionWaypointTask waypoint_task
---
bool result
